2016/presentations/inteligentny_dom_z_(micro)python/text.md

```diff
@@ -0,0 +1,89 @@
+
+Ale bez obaw, stworzenie urządzenia elektronicznego przestało wiązać się z niskopoziomowym programowaniem mikrokontrolera i studiowaniem not aplikacyjnych układów scalonych. Silnie rozwijający się rynek IoT (Internet of Things) oraz platform prototypowych daje do dyspozycji cały szereg modułów elektronicznych gotowych do wykorzystania bez wgłębiania się w ich budowę, a nowoczesne narzędzia deweloperskie pozwalają nam na napisanie własnego systemu wbudowanego w języku wysokiego poziomu. Niemniej takie podejście sprawdzi się tylko na początku, dlatego zachęcam do pogłębiania swojej wiedzy w dziedzinie elektroniki. Interakcja ze sprzętem daje wiele radości, a uruchomione urządzenie potrafi przynieść wymierne korzyści.
+
+W celu rozpoczęcia pracy niezbędne jest zaopatrzenie się w kilka urządzeń oraz modułów, które znajdą zastosowanie w każdej instalacji inteligentnego budynku:
+* Raspberry Pi - centrum zarządzania budynkiem,
+* Moduły ESP-12F - bezprzewodowa komunikacja z czujnikami i urządzeniami wykonawczymi,
+* Konwerter USB-UART - komunikacja przez port szeregowy,
+* Moduł przekaźnika RM0 - umożliwia włączanie/wyłączanie urządzeń 230V,
+* Moduł DS18b20 - termometr 1-wire,
+* Zasilacz 5V oraz 3.3V.
+
+Już tak krótka lista pozwoli na sterowanie oświetleniem czy sprawowanie kontroli nad temperaturą w domu. Na początku, do testowania funkcjonalności nowych modułów elektronicznych, można wykorzystać jedną z platform prototypowych np. Arduino.
+
+## MicroPython i moduł ESP8266
+MicroPython to implementacja języka Python 3, która zawiera niewielki podzbiór biblioteki standardowej języka Python i jest zoptymalizowana pod kątem działania na mikrokontrolerach [micropython]. Istnieje kilka platform sprzętowych pozwalających na uruchomienie środowiska MicroPython takich jak pyBoard czy WiPy. Układem, na którym oprzemy naszą instalację inteligentnego domu będzie ESP8266 [esp8266], a konkretnie moduł o nazwie ESP-12F.
+Co oferuje nam urządzenie wielkości znaczka pocztowego (24 x 16 mm) za niecałe 2 dolary? Oto podstawowe cechy modułu ESP-12F:
+- 32 bit RISC CPU - Tensilica Xtensa LX106 @80 MHz,
+- 11 wyprowadzeń GPIO - wejścia/wyjścia cyfrowe,
+- 1 wejście ADC - 10 bitowy przetwornik analogowo-cyfrowy,
+- interfejsy - SPI, I2C, 2 x UART,
+- WiFi 802.11 b/g/n (2.4 GHz),
+- 4 MB flash.
+
+Komunikacja z modułem ESP-12F odbywa się za pomocą portu szeregowego. Konwerter USB-UART należy podłączyć do wyprowadzeń oznaczonych jako TXD oraz RXD. Trzeba zwrócić uwagę, że komponent zasilany jest napięciem 3.3 V i pracuje na logice o tym samym potencjale. Do zasilania układu należy wykorzystać źródło napięcia o wydajności około 300 mA. W celu uruchomienia modułu wymagane jest również podciągnięcie pinu EN (CH_PD) do VCC oraz pinu GPIO15 do GND przez rezystory 10 kOm.
```

Comment:
A jak już na to patrzymy, to czy w języku polskim nie używa się do zapisu ułamków dziesiętnych przecinka, a nie kropki?